apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerDbType.ValueConversion.cs

```diff
@@ -198,6 +199,17 @@ internal Value ToProtobufValue(object value, SpannerConversionOptions options)
                     }
                     throw new ArgumentException("Struct parameters must be of type SpannerStruct");
 
+                case TypeCode.Numeric:
+                    if (value is SpannerNumeric spannerNumeric)
+                    {
+                        return new Value {StringValue = spannerNumeric.ToString()};
```

Comment:
Consider using the `Value.ForString` helper method, designed to make `Value` easier to work with.

Reply:
That's much nicer. Thank you.